<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefDlg</class>
 <widget class="QDialog" name="RefDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabRef">
      <attribute name="title">
       <string>Reference</string>
      </attribute>
      <widget class="SetTextComboBox" name="comboType">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>111</width>
         <height>26</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>unknown</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>journal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>inproceedings</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>conference</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>technical report</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>thesis</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="labelType">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabSource">
      <attribute name="title">
       <string>BibTex source</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SetTextComboBox</class>
   <extends>QComboBox</extends>
   <header>SetTextComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RefDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RefDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
